### PR TITLE
add-kir-dev-20+-post

### PR DESCRIPTION
**Pull request by SepsiLaszlo**
## What is new?

- Kir-Dev 20 blog post with invitation, form and a exciting program

### CODE REVIEW

File: src/content/posts/2022-05-08-kir-dev-20.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+```toc
+# There comes the toc
+```
+
+## Megrendezésre kerül a Kir-Dev 20+ !
+
+![kir-dev-at-konf](https://warp.sch.bme.hu/img/blobs/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbElCIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--37cb6341ad70d92d1a0dc938363cd278911931fa/kir-dev3.jpg)
+
+A Kir-Dev 2021-ben elérte működésének 20-adik évfordulóját. Ennek a kiemelkedő mérföldkőnek a megünneplését nem szeretnénk tovább húzni. Igy idén nyáron, a Kir-Dev 20+ rendezvény formájában szeretnénk megünnepelni ezt, a kör valamennyi tagjával és öregtagjával együtt!
```
Comment (DannySS123) [inline src/content/posts/2022-05-08-kir-dev-20.md:23]: Igy idén nyáron -> Így idén nyáron

File: src/content/posts/2022-05-08-kir-dev-20.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+A rendezvényre több programmal is készülünk a résztvevők számára:
+
+- Megnyitó
+
+  - Résztvevők köszöntése, valamint egy tömör összefoglaló a Kir-Dev utóbbi néhány évéről, aktuális projektjeiről.
+
+- Tagok bemutatkozása
+
+  - A program ezen a részén felkérjük a megjelent tagokat és öregtagokat, hogy mutatkozzanak be és néhány mondatban foglalják össze, körön belüli és körön túli tevékenységüket.
```
Comment (DannySS123) [inline src/content/posts/2022-05-08-kir-dev-20.md:44]: tevékenységüket -> tevékenységeiket

**Review by DannySS123 — APPROVED** [review]
Great work, looking very promising! 💯 🔥 
Just some small grammar corrections.

**Comment by tomitheninja** [comment]
I think the space between the plus and the exclamation sign looks weird.
image

**Comment by tomitheninja** [comment]
The site here feels skinny for me.

Can we replace this image with a gallery?
I think between a carousel and a full screen image scrolling system like on https://spot.sch.bme.hu

image

**Review by bbucsy — APPROVED** [review]
Nice!

**Comment by SepsiLaszlo** [comment]
@DannySS123 thank you for the suggestions, I corrected the typos!

**Review by DannySS123 — APPROVED** [review]

**Comment by SepsiLaszlo** [comment]
@tomitheninja I don't think there is an easy way to do what you suggested and the other posts have the same design. 
So I think we should keep this as is, and look into the possible improvemnets in the future.

File: src/content/posts/2022-05-08-kir-dev-20.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+## Megrendezésre kerül a Kir-Dev 20+!
+![kir-dev-at-konf](https://warp.sch.bme.hu/img/blobs/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbElCIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--37cb6341ad70d92d1a0dc938363cd278911931fa/kir-dev3.jpg)
+
+A Kir-Dev 2021-ben elérte működésének 20-adik évfordulóját. Ennek a kiemelkedő mérföldkőnek a megünneplését nem szeretnénk tovább húzni. Így idén nyáron, a Kir-Dev 20+ rendezvény formájában szeretnénk megünnepelni ezt, a kör valamennyi tagjával és öregtagjával együtt!
+
+## Jelentkezés
+
+![kir-dev-join-us](https://warp.sch.bme.hu/img/blobs/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbEVCIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--9dd76b2443b2dc393898c58c6550fb7c29c857ea/kir-dev2.jpg)
+
+A rendezvény a Schönherz Kollégiumban kerül megrendezésre, 2022-es nyár folyamán. A pontos időpontot szeretnék úgy megválasztani, hogy minél több öregtagunk el tudjon jönni. Ha körünk öregtaja vagy, akkor az alábbi form-on tudod jelezni, hogy mely időpontok alkalmasak számodra.
```
Comment (triszt4n) [inline src/content/posts/2022-05-08-kir-dev-20.md:28]: öregtaja -> öregtagja

File: src/content/posts/2022-05-08-kir-dev-20.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+https://forms.gle/XjYcb3UKZ6RzQkR86
```
Comment (triszt4n) [inline src/content/posts/2022-05-08-kir-dev-20.md:29]: The form requires logging in with Google account. Is it necessary? We also require their email address, isn't that enough?

**Review by triszt4n — CHANGES_REQUESTED** [review]
Nicely written and designed post! Please see my requests. Also: we might want to tell them, that they can ask about anything via our email address or Slack channels.

**Comment by triszt4n** [comment]
> The site here feels skinny for me.
> 
> Can we replace this image with a gallery?
> I think between a carousel and a full screen image scrolling system like on https://spot.sch.bme.hu
> 
> image
> 

@tomitheninja let's not redesign the site for now, but send an issue to the repo with your requests, thx

**Comment by balintkiraly** [comment]
Maybe a few words about Schonherz's current COVID restrictions would be great for the old members (I think this can be placed in the form).
And a feedback for the form.. the question about the days should be multi-select

**Comment by SepsiLaszlo** [comment]
@balintkiraly Thank you for the feedback, I made the suggested change!

File: src/content/posts/2022-05-08-kir-dev-20.md (hunk 4 above)
Comment (SepsiLaszlo) [reply src/content/posts/2022-05-08-kir-dev-20.md:30]: @triszt4n Thank you for the sharp review! :eagle:  :detective: 
I made the suggested changes!

File: src/content/posts/2022-05-08-kir-dev-20.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+author: sepsilaci
+date: 2022-05-19 21:02:53
+title: Kir-Dev 20+!
+lead: Megrendezésre ketül a Kir-Dev 20!
```
Comment (triszt4n) [inline src/content/posts/2022-05-08-kir-dev-20.md:6]:
```suggestion
lead: Megrendezésre kerül a Kir-Dev 20!
```

**Review by triszt4n — CHANGES_REQUESTED** [review]

**Review by triszt4n — APPROVED** [review]
All nice and good

**Review by DannySS123 — APPROVED** [review]
👌

**Review by bbucsy — APPROVED** [review]

**Pull request merged**